packages/hop-node/src/watchers/BondWithdrawalWatcher.ts

```diff
@@ -121,13 +112,28 @@ class BondWithdrawalWatcher extends BaseWatcher {
       return
     }
 
-    const availableCredit = await destBridge.getAvailableCredit()
+    let availableCredit = await destBridge.getAvailableCredit()
+    const includePendingAmount = ([Chain.Optimism, Chain.Arbitrum] as string[]).includes(destinationChain)
+    if (includePendingAmount) {
+      const pendingAmount = await sourceL2Bridge.getPendingAmountForChainId(destinationChainId)
+      availableCredit = availableCredit.sub(pendingAmount).sub(amount)
```

Comment:
1. Why is the `2*transferAmount` not included here?
2. I thought we were going to get this from an sdk call so that this piece of code was only defined once but used on both the FE/SDK and here so we don't get out of sync. I might have misunderstood.

Reply:
1. Fixed
2. tried importing the sdk but didn't seem right because it would be basically reinitializing the contracts and providers which can complicate things (especially since the sdk doesn't have retry decorators) for this single check. If the sdk was more robust then it would make more sense I think.

Reply:
2. That SGTM & I agree w/ your analysis